### ISSUE TITLE
Escaping class names with square brackets and other unusual characters

### ISSUE THREAD

**Issue by SHxKM** [bigskysoftware/_hyperscript#376]
I've got a class like: `bg-rose-50/[87.5%]` (from the very popular Tailwind CSS framework).

When I try to do:

```
on click remove .bg-rose-50/[87.5%] from #some_div then...
```

hyperscript complains:

> Unexpected Token : /

 Is there anyway to escape characters like `/[]` (I'm guessing that is what's tripping hyperscript)

**Comment by dz4k** [comment]
```
set class to "bg-rose-[87.5%]"
remove .{class} from #some_div
```

Or more tersely:

```
remove .{"bg-rose-[87.5%]"} from #some_div
```

**Comment by SHxKM** [comment]
@dz4k worked beautifully, thank you! 

I wonder if this also resolves #282...?
